classroom-dashboard/app/controllers/sections_controller.rb

```diff
@@ -1,70 +1,69 @@
 class SectionsController < ApplicationController
+  
+  # Find the section before any other action
+  before_action :find_section, only: [:show, :edit, :update, :destroy, :flashcards]
 
-	# Find the section before any other action
-	before_action :find_section, only: [:show, :edit, :update, :destroy, :flashcards]
+  # Order in which the sections are listed on the home page
+  def index
+    @section = Section.all.order("semester DESC")
+  end
+	
+  def show
+  end
 
-	# Order in which the sections are listed on the home page
-	def index
-		@section = Section.all.order("semester DESC")
-	end
+  # Create a new Section
+  def new
+    @section = Section.new
+  end
 
-	def show
-	end
+  def edit
+    @Student = Student.find(params[:id])
+  end
 
-	# Create a new Section
-	def new
-		@section = Section.new
-	end
+  # when /flashcards page loads, a random student is loaded into @student
+  def flashcards
```

Comment:
Is this function going to be implemented or does this empty function do what it is supposed to? If the later I don't know if it is good form to rely on empty functions doing things. If it is the former, perhaps update that this is to be completed.